acquisitions/src/main/resources/thunderjet/cross-modules/features/rollover-with-no-settings.feature

```diff
@@ -0,0 +1,162 @@
+# For MODORDERS-1162
+Feature: Rollover with no settings
+
+  Background:
+    * print karate.info.scenarioName
+    * url baseUrl
+    * callonce login testAdmin
+    * def okapitokenAdmin = okapitoken
+
+    * call loginRegularUser testUser
+    * def okapitokenUser = okapitoken
+
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json, text/plain' }
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json, text/plain' }
+
+    * configure headers = headersUser
+
+    * callonce variables
+
+    * def createFiscalYear = read('classpath:thunderjet/mod-finance/reusable/createFiscalYear.feature')
+    * def createOrder = read('classpath:thunderjet/mod-orders/reusable/create-order.feature')
+    * def createOrderLine = read('classpath:thunderjet/mod-orders/reusable/create-order-line.feature')
+    * def openOrder = read('classpath:thunderjet/mod-orders/reusable/open-order.feature')
+    * def validateOrderLineEncumbranceLinks = read('classpath:thunderjet/mod-orders/reusable/validate-order-line-encumbrance-links.feature')
+
+  @Positive
+  Scenario: Rollover with no settings
+    * def fromYear = call getCurrentYear
+    * def toYear = parseInt(fromYear) + 1
+    * def fiscalYearId1 = call uuid
+    * def fiscalYearId2 = call uuid
+    * def ledgerId = call uuid
+    * def fundId = call uuid
+    * def budgetId1 = call uuid
+    * def budgetId2 = call uuid
+
+    * def orderId1 = call uuid
+    * def orderId2 = call uuid
+    * def orderId3 = call uuid
+    * def orderId4 = call uuid
+
+    * def poLineId1 = call uuid
+    * def poLineId2 = call uuid
+    * def poLineId3 = call uuid
+    * def poLineId4 = call uuid
+
+    * def ongoingObj = { "interval": 123, "isSubscription": true, "renewalDate": "2022-05-08T00:00:00.000+00:00" }
+
+    * def rolloverId = call uuid
+
+    * table orders
+      | id       | orderId  | orderType  | reEncumber | ongoing    |
+      | orderId1 | orderId1 | 'One-Time' | true       | null       |
+      | orderId2 | orderId2 | 'One-Time' | false      | null       |
+      | orderId3 | orderId3 | 'Ongoing'  | true       | ongoingObj |
+      | orderId4 | orderId4 | 'Ongoing'  | false      | ongoingObj |
+
+    * table orderLines
+      | id        | orderId  | listUnitPrice |
+      | poLineId1 | orderId1 | 10.0          |
+      | poLineId2 | orderId2 | 10.0          |
+      | poLineId3 | orderId3 | 10.0          |
+      | poLineId4 | orderId4 | 10.0          |
```

Comment:
Suggestion: I suggest moving to these close to creation of order and orderline, so it will be easier to find these while reading scenario